facet-toml/src/serialize.rs

```diff
@@ -0,0 +1,123 @@
+//! Create and/or write TOML strings from Rust values.
+
+use core::fmt::Result;
+
+#[cfg(feature = "alloc")]
+use alloc::string::String;
+
+use facet_core::{Def, Facet, FieldAttribute};
+use facet_reflect::{Peek, ScalarType};
+use toml_write::TomlWrite;
+
+/// Implement writing TOML values to types that allow being written to.
+pub trait TomlSerialize: TomlWrite {
```

Comment:
I see little value in putting `TomlWrite` in the public API or in offering this through an extension trait

Reply:
I thought it would be useful so that it is automatically implemented for all types that `TomlWrite` also implements. But since the types are fairly restricted, maybe there's indeed not much value.